roles/contiv_network/tasks/ovs.yml

```diff
@@ -53,9 +53,16 @@
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
 
+- name: check selinux status
+  shell: sestatus | grep "disabled"
```

Comment:
@gaurav-dalvi 

On my machine I see enabled at two lines in the command output, I think this check needs to be more specific

Can we try something like this:

```
shell: sestatus | egrep "SELinux status:.*enabled"
```

```
[vagrant@cluster-node1 ~]$ sestatus
SELinux status:                 enabled            <<<<<
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   permissive
Mode from config file:          permissive
Policy MLS status:              enabled             <<<<<<<
Policy deny_unknown status:     allowed
Max kernel policy version:      28 
```